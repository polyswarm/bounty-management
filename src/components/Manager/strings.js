const strings = {
  create: 'Create Bounty',
};
export default strings;
