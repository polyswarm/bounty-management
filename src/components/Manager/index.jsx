// Vendor imports
import React, { Component } from 'react';
import PropTypes from 'prop-types';
// Component Imports
import strings from './strings';
import './styles.css';

class Manager extends Component {

  render() {
    return (
      <div>
        <a href='/create'>{strings.create}</a>
      </div>
    );
  }

}
Manager.defaultProps = {
  bounties: PropTypes.array.isRequired,
};
export default Manager;
