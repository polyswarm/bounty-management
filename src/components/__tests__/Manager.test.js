import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import {BrowserRouter} from 'react-router-dom';
import Manager from '../Manager';

it('renders without crashing', () => {
  const wrapper = render(
    <BrowserRouter>
      <Manager />
    </BrowserRouter>
  );
  expect(renderToJson(wrapper)).toMatchSnapshot();
});
