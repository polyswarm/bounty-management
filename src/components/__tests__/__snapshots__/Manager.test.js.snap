// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <a
    href="/create"
  >
    Create Bounty
  </a>
</div>
`;
